src/lib/blueprint.ts

```diff
@@ -117,14 +121,327 @@ interface ObjectProperty extends BaseProperty {
+        : '',
+    isUndocumented: false,
+    isDeprecated: false,
+    deprecationMessage: '',
+    parameters: createParameters(operation),
+    request: createRequest(method, operation),
+    response: createResponse(
+      'responses' in operation ? operation.responses : {},
+    ),
+  }
+}
+
+const createParameters = (operation: OpenAPIOperation): Parameter[] => {
+  return 'parameters' in operation && Array.isArray(operation.parameters)
+    ? operation.parameters
+        .filter((param) => typeof param === 'object' && param !== null)
+        .map(createParameter)
+    : []
+}
+
+const createParameter = (param: OpenAPIParameter): Parameter => {
+  return {
+    name: 'name' in param && typeof param.name === 'string' ? param.name : '',
+    isRequired:
+      'required' in param && typeof param.required === 'boolean'
+        ? param.required
+        : false,
+    isUndocumented: false,
+    isDeprecated: false,
+    deprecationMessage: '',
+    description:
+      'description' in param && typeof param.description === 'string'
+        ? param.description
+        : '',
+  }
+}
+
+const createRequest = (
+  method: string,
+  operation: OpenAPIOperation,
+): Request => {
+  const uppercaseMethod = openapiMethodToMethod(
+    method.toLowerCase() as LowercaseMethod,
+  )
+
+  return {
+    methods: [uppercaseMethod],
+    semanticMethod: uppercaseMethod,
+    preferredMethod: uppercaseMethod,
+    parameters: createParameters(operation),
+  }
+}
+
+const createResources = (
+  schemas: OpenAPI['components']['schemas'],
+  isFakeData: boolean,
+  targetSchema: string,
+): Record<string, Resource> => {
+  return Object.entries(schemas)
+    .filter(([schemaName]) => isFakeData || schemaName === targetSchema)
+    .reduce<Record<string, Resource>>((acc, [schemaName, schema]) => {
+      if (
+        typeof schema === 'object' &&
+        schema !== null &&
+        'properties' in schema &&
+        typeof schema.properties === 'object' &&
+        schema.properties !== null
+      ) {
+        acc[schemaName] = {
+          resourceType: schemaName,
+          properties: createProperties(schema.properties),
+        }
+      }
+      return acc
+    }, {})
+}
+
+const createResponse = (responses: OpenAPIOperation['responses']): Response => {
+  if (typeof responses !== 'object' || responses === null) {
+    return { responseType: 'void', description: 'No description available' }
+  }
+
+  const okResponse = responses['200']
+  if (typeof okResponse !== 'object' || okResponse === null) {
+    return { responseType: 'void', description: 'No description available' }
+  }
+
+  const content = 'content' in okResponse ? okResponse.content : null
+  if (typeof content !== 'object' || content === null) {
+    return {
+      responseType: 'void',
+      description:
+        'description' in okResponse &&
+        typeof okResponse.description === 'string'
+          ? okResponse.description
+          : 'No description available',
```

Comment:
We should still default to `''` if there is no description. As a followup PR, we can add a `logEmptyPropertyWarning` function for all of these cases.